### DISCUSSION TITLE
Inappropriate example screenshot on the main site

### DISCUSSION THREAD

**Discussion by Cwpute** [aclap-dev/video-downloadhelper#171] [Help and Q&A]
### Your question

Hi,
I've been a long-time DH user and have used your tool for various useful pruposes. I purchased a license several years ago and am still using it, even after having entirely changed computer 3 times already ;) Thanks a whole lot !

I'm reaching to you about an issue i have with your website's landing page.
It features a screenshot showcasing how DownloadHelper looks and works. It's quite fine and serviceable, until you notice what type of video it is downloading.
![inappropriate screeenshot](https://www.downloadhelper.net/images/screenshot.png)
I can see why it is funny displaying such a video being downloaded, given the range of websites DownloadHelper supports 👀 But there is a difference between exciting videos featuring willing adults, and stolen sights of an unwilling public personality. It comes off as creepy and unrespectful.

Would you consider remaking that screenshot example with other kind of videos being downloaded ?
This could be an opportunity to showcase content creators you like, ones that can dabble into adult content even (it still makes for a funny nod on the main page for this addon).

### Addon & CoApp version and details

(irrelevant)

[the remark this answers is not in the document]
**Reply by jcfrog** [reply]
yep, that's not a bad idea :)
One thing is for sure, this home page will change soon.

**Comment by jcfrog** [comment]
One of these images could do the trick while we wait for a more complete overhaul of the site, don't you think?
https://github.com/aclap-dev/video-downloadhelper/wiki/Basic-Usage

**Comment by Cwpute** [comment]
Would be perfectly fine, as far as i'm concerned 👍

Le 19 février 2024 15:33:37 GMT+01:00, jcfrog ***@***.***> a écrit :
